[2024-05-06 07:43:13,538] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-05-05T00:00:00+00:00 [queued]>
[2024-05-06 07:43:13,563] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-05-05T00:00:00+00:00 [queued]>
[2024-05-06 07:43:13,564] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-06 07:43:13,564] {taskinstance.py:1043} INFO - Starting attempt 2 of 6
[2024-05-06 07:43:13,565] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-06 07:43:13,579] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): poastgres_to_s3> on 2024-05-05T00:00:00+00:00
[2024-05-06 07:43:13,585] {standard_task_runner.py:52} INFO - Started process 301 to run task
[2024-05-06 07:43:13,590] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_prostgres_hooks_v2', 'poastgres_to_s3', '2024-05-05T00:00:00+00:00', '--job-id', '137', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpthlqui2j', '--error-file', '/tmp/tmp9u_x75dk']
[2024-05-06 07:43:13,595] {standard_task_runner.py:77} INFO - Job 137: Subtask poastgres_to_s3
[2024-05-06 07:43:13,657] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-05-05T00:00:00+00:00 [running]> on host d4a0c764ecae
[2024-05-06 07:43:13,745] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mark
AIRFLOW_CTX_DAG_ID=dag_with_prostgres_hooks_v2
AIRFLOW_CTX_TASK_ID=poastgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2024-05-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-05T00:00:00+00:00
[2024-05-06 07:43:13,767] {base.py:74} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: test, Login: airflow, Password: XXXXXXXX, extra: None
[2024-05-06 07:43:13,824] {dag_with_postgres_hooks.py:32} INFO - Saved orders data in text file: dags/get_orders_20240505.txt
[2024-05-06 07:43:13,828] {base_aws.py:362} INFO - Airflow Connection: aws_conn_id=minio_conn
[2024-05-06 07:43:13,837] {base_aws.py:164} INFO - Credentials retrieved from extra_config
[2024-05-06 07:43:13,837] {base_aws.py:81} INFO - Creating session with aws_access_key_id=ROOTUSER region_name=None
[2024-05-06 07:43:13,856] {base_aws.py:151} INFO - role_arn is None
[2024-05-06 07:43:14,064] {dag_with_postgres_hooks.py:41} INFO - Orders file /tmp/tmp2jby4ww320240505 has been pushed to S3!
[2024-05-06 07:43:14,072] {python.py:118} INFO - Done. Returned value was: None
[2024-05-06 07:43:14,087] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_prostgres_hooks_v2, task_id=poastgres_to_s3, execution_date=20240505T000000, start_date=20240506T074313, end_date=20240506T074314
[2024-05-06 07:43:14,123] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-06 07:43:14,170] {local_task_job.py:146} INFO - Task exited with return code 0
